Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'puma', '~> 5.0'
 # Use Active Storage variant
 # gem 'image_processing', '~> 1.2'
 gem 'devise', '~> 4.8.0'
-gem 'devise_token_auth', '~> 1.1.5'
+gem 'devise_token_auth', '~> 1.2.0'
```

Comment:
Kudos for this initiative. 👏 
It's important to not use deprecated stuff, but often we ignoring that kind of message.